client/src/components/GasStationDetail.jsx

```diff
@@ -129,7 +129,7 @@ const InactiveStar = styled(MdStarOutline)`
   color: #2196f3;
 `;
 
-const OpeningTimesWrapper = styled.p`
+const OpeningTimesWrapper = styled.ul`
```

Comment:
If you change this into a `ul`, the children should be `li` elements; but as far as I can see, `OpeningTimes` is not such a list item, is it?